### ISSUE TITLE
Aegis import fails when any entries use an SVG icon

### ISSUE THREAD

**Issue by Daniel15** [jamie-mh/AuthenticatorPro#267]
**Describe the bug**
I've tried importing from Aegis using the unencrypted JSON format, and it says "the file could not be imported". 

I do have icons for a lot of them, from the aegis-icons pack (https://github.com/aegis-icons/aegis-icons), and from the stack trace I collected via `logcat` it appears that's the issue?

```
09-11 15:26:23.701  5882  5882 E AUTHPRO : System.Exception: Image could not be loaded.
09-11 15:26:23.701  5882  5882 E AUTHPRO :   at AuthenticatorPro.Droid.Data.CustomIconDecoder.Decode (System.Byte[] rawData) [0x000ac] in <002f128ee3b84bb487b4f7aaddf6fd65>:0
09-11 15:26:23.701  5882  5882 E AUTHPRO :   at AuthenticatorPro.Shared.Data.Backup.Converter.AegisBackupConverter.Convert (System.Byte[] data, System.String password) [0x001de] in <636461b02ef549efba58d3344ceb87d6>:0
09-11 15:26:23.701  5882  5882 E AUTHPRO :   at AuthenticatorPro.Droid.Activity.MainActivity+<>c__DisplayClass87_0.<ImportFromUri>g__ConvertAndRestore|0 (System.String password) [0x00088] in <002f128ee3b84bb487b4f7aaddf6fd65>:0
09-11 15:26:23.701  5882  5882 E AUTHPRO :   at AuthenticatorPro.Droid.Activity.MainActivity.ImportFromUri (AuthenticatorPro.Shared.Data.Backup.Converter.BackupConverter converter, Android.Net.Uri uri) [0x00163] in <002f128ee3b84bb487b4f7aaddf6fd65>:0
```

The problem is that I have maybe over 50 accounts in Aegis so I'm not sure which ones are causing the issue.

Edit: So as far as I can tell, Authenticator Pro's Aegis import only supports bitmaps. It uses Android's `BitmapFactory`:
https://github.com/jamie-mh/AuthenticatorPro/blob/5c3aae02bea49ae2d1ccff4d600296c490d04b60/AuthenticatorPro.Droid/src/CustomIconDecoder.cs#L111

which only supports bitmaps and has no ability to rasterize vector types: 
https://github.com/aosp-mirror/platform_frameworks_base/blob/master/libs/hwui/jni/BitmapFactory.cpp#L57-L76

However, Aegis 2.0 prefers SVG icons since they're scalable and have a smaller file size. In general, SVGs work very well for logos as they tend to be simple shapes and colours.

I have 38 icons of type `image/svg+xml` in my Aegis export. I expect that deleting those icons will fix the import.

**To Reproduce**
1. Export from Aegis
2. Import into Authenticator Pro
 
**Expected behavior**
Import works.

**App Version**
1.14.10

**Comment by Daniel15** [comment]
So as far as I can tell, Authenticator Pro's Aegis import only supports bitmaps. It uses `BitmapFactory`:
https://github.com/jamie-mh/AuthenticatorPro/blob/5c3aae02bea49ae2d1ccff4d600296c490d04b60/AuthenticatorPro.Droid/src/CustomIconDecoder.cs#L111

which only supports bitmaps and has no ability to rasterize vector types: 
https://github.com/aosp-mirror/platform_frameworks_base/blob/master/libs/hwui/jni/BitmapFactory.cpp#L57-L76

However, Aegis 2.0 prefers SVG icons since they're scalable and have a smaller file size. In general, SVGs work very well for logos as they tend to be simple shapes and colours.

I have 38 icons of type `image/svg+xml` in my Aegis export. I expect that deleting those icons will fix the import.

**Comment by jamie-mh** [comment]
Thanks for the investigation. Indeed this seems like an oversight.
I don't want to support SVG icons (#269), but they shouldn't prevent the import from working correctly. I will fix this.

**Comment by jamie-mh** [comment]
Fixed in 1.15.0
